ethereum/contract_addresses.go

```diff
@@ -22,6 +22,22 @@ type ContractAddresses struct {
 	ChaiToken           common.Address `json:"chaiToken"`
 }
 
+func GanacheAddresses() ContractAddresses {
```

Comment:
Sorry for not catching this last time, but after seeing this I realized we can simplify this even further. We don't really need this to be function now, it can just be a `var`.

```suggestion
var GanacheAddresses = ContractAddresses {
```

Reply:
Yeah I also thought of that. Is there a way of making variables readonly in Golang? I think that's really what we want here